demos/Ollama Chat Service/ollama.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2024 Inclusive Design Research Centre, OCAD University
+ * All rights reserved.
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/inclusive-design/adaptive-palette/blob/main/LICENSE
+ */
+
+import ollama from "ollama/browser";
+
+// Handle click on "Ask" buttons
+function askClicked(event) {
+  const singleSentence = "Answer with a single grammatically correct sentence.";
+  // Empty out the response area
+  document.getElementById("ollamaOutput").innerText = "Working...";
+  if (event.target.id === "singleSentence") {
+    executeAsk(singleSentence);
+  }
+  else {
+    executeAsk();
+  }
+}
+
+// Function for passing the chat prompt using ollama service.
+async function queryChat (query) {
+  const message = { role: "user", content: query };
+  const response = await ollama.chat({
+    model: "phi3",
```

Comment:
It might be more adaptable to define the model as a variable at the beginning of the script. The "phi-3" portion in the heading `<h1>Experimenting with Ollama (phi-3) in a Browser</h1>` could be adjusted based on the model value. Moreover, update the README to describe where to adjust this variable.

This will make it easier for trying different models.